webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -221,28 +223,31 @@ const JobQueue: React.FC<Props> = ({ selectedRp, jobState }) => {
           case 'actions':
             return {
               ...col,
-              render: (_, record) => (
-                <div>
-                  <ActionDropdown<JobAction>
-                    actionOrder={[
-                      JobAction.ManageJob,
-                      JobAction.MoveToTop,
-                      JobAction.ViewLog,
-                      JobAction.Cancel,
-                      JobAction.Kill,
-                    ]}
-                    confirmations={{
-                      [JobAction.Cancel]: { cancelText: 'Abort', onError: handleError },
-                      [JobAction.Kill]: { danger: true, onError: handleError },
-                      [JobAction.MoveToTop]: { onError: handleError },
-                    }}
-                    id={record.name}
-                    kind="job"
-                    onError={handleError}
-                    onTrigger={dropDownOnTrigger(record)}
-                  />
-                </div>
-              ),
+              render: (_, record) => {
+                if (!('entityId' in record)) return;
+                return (
```

Comment:
couldn't find a way to avoid the indentation w/o the linter complaining

Reply:
what you have is clearer but I think it can be done like this:
```
render: (_, record) => 'entityId' in record ? (
  ...JSX...
) : null);
```